### PR TITLE
POC import cleanup 

### PR DESCRIPTION
**Pull request by wouterdb**
# Description

* Short description here *

closes *Add ticket reference here*

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [ ] Attached issue to pull request
- [ ] Changelog entry
- [ ] Type annotations are present
- [ ] Code is clear and sufficiently documented
- [ ] No (preventable) type errors (check using make mypy or make mypy-diff)
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )
- [ ] If this PR fixes a race condition in the test suite, also push the fix to the relevant stable branche(s) (see [test-fixes](https://internal.inmanta.com/development/core/tasks/build-master.html#test-fixes) for more info)

### CODE REVIEW

File: src/inmanta/plugins.py [hunk 1]
```diff
@@ -21,40 +21,35 @@
 import inspect
 import os
 import subprocess
+import typing
 import warnings
-from collections import abc
-from typing import TYPE_CHECKING, Any, Callable, Literal, Optional, Type, TypeVar
 
+import inmanta.ast as ast
```
Comment (wouterdb) [inline src/inmanta/plugins.py:27]: I like this to keep them short (i.e. snip off as much as possible)

File: src/inmanta/plugins.py [hunk 2]
```diff
@@ -21,40 +21,35 @@
...
-from inmanta import const, protocol, util
-from inmanta.ast import LocatableString, Location, Namespace, Range, RuntimeException, TypeNotFoundException, WithComment
-from inmanta.ast.type import NamedType
-from inmanta.config import Config
-from inmanta.execute.proxy import DynamicProxy
-from inmanta.execute.runtime import QueueScheduler, Resolver, ResultVariable
-from inmanta.execute.util import NoneValue, Unknown
-from inmanta.stable_api import stable_api
-from inmanta.warnings import InmantaWarning
+import inmanta.warnings
```
Comment (wouterdb) [inline src/inmanta/plugins.py:29]: This one can't be shortened, as we also import warnings: keep it like this or make it into inmanta_warnings?

File: src/inmanta/plugins.py [hunk 3]
```diff
@@ -21,40 +21,35 @@
...
+from inmanta import config, const, protocol, stable_api, util
+from inmanta.execute import proxy, runtime
+from inmanta.execute import util as execute_util
```
Comment (wouterdb) [inline src/inmanta/plugins.py:32]: this one shadows `inmanta.util` how to best handle this?

File: src/inmanta/plugins.py [hunk 4]
```diff
@@ -21,40 +21,35 @@
 import inspect
 import os
 import subprocess
+import typing
 import warnings
-from collections import abc
```
Comment (wouterdb) [inline src/inmanta/plugins.py:25]: Do we want just `abc` or `collection.abc`

File: src/inmanta/plugins.py (hunk 1 above)
Comment (arnaudsjs) [reply src/inmanta/plugins.py:27]: I like this too

File: src/inmanta/plugins.py (hunk 2 above)
Comment (arnaudsjs) [reply src/inmanta/plugins.py:29]: By making it into `inmanta_warnings` we don't win anything, right? In that case I would keep it like this.

File: src/inmanta/plugins.py (hunk 3 above)
Comment (arnaudsjs) [reply src/inmanta/plugins.py:32]: I like it like this were we alias to `execute_util`.

File: src/inmanta/plugins.py (hunk 4 above)
Comment (arnaudsjs) [reply src/inmanta/plugins.py:25]: I don't have a strong opinion on this.

**Pull request closed without merging**